micall/tests/test_plot_contigs.py

```diff
@@ -1196,3 +1196,84 @@ def test_match_g2p_fake_contig():
     assert contig_matcher.ref == contig_name
     assert contig_matcher.num is None
     assert contig_matcher.name == contig_name
+
+
+def test_plot_genome_concordance():
+    genome_coverage_csv = StringIO("""\
+contig,coordinates,query_nuc_pos,refseq_nuc_pos,ins,dels,coverage,concordance
+1-HCV-1a,HCV1A,1,1,0,0,5,0
+1-HCV-1a,HCV1A,2,2,0,0,5,0.25
+1-HCV-1a,HCV1A,3,3,0,0,7,0.5
+1-HCV-1a,HCV1A,4,4,0,0,5,1
+1-HCV-1a,HCV1A,5,5,0,0,5,0.3
+1-HCV-1a,HCV1A,6,6,0,0,5,0
+""")
+    expected_figure = """\
+5'[1-341], C[342-914], E1[915-1490], E2[1491-2579], p7[2580-2768], \
+NS2[2769-3419], NS3[3420-5312], NS4b[5475-6257], NS4a[5313-5474], \
+NS5a[6258-7601], NS5b[7602-9377], 3'[9378-9646]
+Coverage 0.0, 25.0, 50.0, 100.0, 30.0, 0.0
```

Comment:
Is it a pain to label this as concordance? It would make the test easier to understand.

Reply:
Done